components/markdown/src/markdown.rs

```diff
@@ -139,7 +139,27 @@ fn fix_link(
                 resolved.permalink
             }
             Err(_) => {
-                return Err(anyhow!("Relative link {} not found.", link));
+                match context.config.link_checker.internal_level {
+                    config::LinkCheckerLevel::ErrorLevel => {
+                        return Err(anyhow!(
+                            "Dead relative link `{}` in {}",
```

Comment:
this one is missing the prefix?
I think it would be easier to add a method to `LinkCheckerLevel` to build the error string.

Reply:
Error messages are prefixed with `Error: ` by anyhow, so they don't need our custom prefix.  Except in the cases where an error message vec is joined with `\n`, then every line except the first one needs `Error: ` prefixed.  It definitely makes handling the messages more awkward and verbose.  If the error/warn messages were identical except for the prefix, the code would be a lot cleaner.  I'm not sure how to achieve that though, with anyhow's automatic prefixing of `Error: `.

Reply:
ah I forgot anyhow was prefixing Error:

Reply:
I'm kinda thinking that the `console::warn` and `console:error` functions should be prefixing `"Warning: "` and `"Error: "` instead of putting that in LinkCheckerLevel.

Reply:
Would make sense imo